tee-worker/ts-tests/examples/direct-invocation/index.ts

```diff
@@ -0,0 +1,96 @@
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+import { TypeRegistry } from '@polkadot/types';
+import { hexToU8a, u8aToHex, u8aConcat } from '@polkadot/util';
+import { teeTypes } from '../../common/type-definitions';
+import {
+    createSignedTrustedCallSetUserShieldingKey,
+    sendRequestFromTrustedCall,
+    getTEEShieldingKey,
+    createSignedTrustedCallCreateIdentity,
+} from './util';
+import { getEnclave, sleep, buildIdentityHelper } from '../../common/utils';
+
+// in order to handle self-signed certificates we need to turn off the validation
+// TODO add self signed certificate
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+const WebSocketAsPromised = require('websocket-as-promised');
+const WebSocket = require('ws');
+const keyring = new Keyring({ type: 'sr25519' });
+
+const PARACHAIN_WS_ENDPINT = 'ws://localhost:9944';
+const WORKER_TRUSTED_WS_ENDPOINT = 'wss://localhost:2000';
```

Comment:
If I understood correctly from Nitesh some time ago, this endpoint is not publicly available for security reasons. Is it still the case?

Reply:
See https://github.com/litentry/litentry-parachain/issues/1010 
We don't see security issues - only server load (similar to any other rpc service provider)